specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/examples/HubVirtualNetworkConnectionUpdateTags.json

```diff
@@ -0,0 +1,59 @@
+{
+    "parameters": {
+        "connectionName": "connection1",
+        "virtualHubName": "virtualHub1",
+        "resourceGroupName": "rg1",
+        "api-version": "2018-12-01",
+        "subscriptionId": "subid",
+        "hubVirtualNetworkConnectionParameters": {
+            "tags": {
+                "key1": "value1",
+                "key2": "value2"
+            }
+        }
+    },
+    "responses": {
+        "200": {
+            "body": [
+                {
+                    "name": "connection1",
+                    "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsofot.Network/virtualHubs/virtualHub1/virtualHubVnetConnections/connection1",
```

Comment:
```suggestion
                    "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/virtualHubs/virtualHub1/virtualHubVnetConnections/connection1",
```

Reply:
Sorry for the typo. Fixed all occurrences.